src/graph/backend/dnnl/kernels/batch_norm.hpp

```diff
@@ -91,6 +91,7 @@ struct batch_norm_fwd_t : public kernel_base_t {
 
     DEF_KERNEL_METHOD_STR(batch_norm_fwd_t)
     DNNL_DISALLOW_COPY_AND_ASSIGN(batch_norm_fwd_t)
+    KERNEL_RESET_ENGINE
```

Comment:
```suggestion
    DEF_KERNEL_METHOD_RESET_ENGINE
```
Just try to align with the name of `DEF_KERNEL_METHOD_STR`.

Reply:
Changed, thanks!